ru-RU/basis.event.md

```diff
@@ -51,7 +53,7 @@ foo.addHandler({
   eventWithArgs: function(sender, a, b){
     // sender -> foo
     // this -> bar
-    console.log('eventWithArgs:', a, b);
+    console.log('sum event in', this.name,'context, result:', a + b);
```

Comment:
Отталкивался от того, что должно быть выведено в консоль(см.ниже)
Но если надо, то могу вернуть
23 мая 2016 г. 3:13 PM пользователь "Roman Dvornov" <
notifications@github.com> написал:

In ru-RU/basis.event.md
<https://github.com/basisjs/articles/pull/21#discussion_r64211669>:

> @@ -51,7 +53,7 @@ foo.addHandler({
>    eventWithArgs: function(sender, a, b){
>      // sender -> foo
>      // this -> bar
> -    console.log('eventWithArgs:', a, b);
> +    console.log('sum event in', this.name,'context, result:', a + b);

Кажется здесь не стоило менять и приводить пример со сложением - пример по
сути абстрактный

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub
<https://github.com/basisjs/articles/pull/21/files/ec42d68f9e59483a8e3252943440a6a6a111f8a4#r64211669>